common/translation.cpp

```diff
@@ -450,6 +454,55 @@ bool TranslationManager::checkHeader(File &in) {
 	return true;
 }
 
+#ifdef USE_FRIBIDI
+String TranslationManager::bidiAlgo(const String input) {
```

Comment:
I am not a big fan of this name. Perhaps `convertBiDiString` could be better?

Reply:
Done